run_testlist.py

```diff
@@ -1033,36 +677,67 @@ def _expand_variant(self, env, ppc, variant):
         self._add_partition(env, nodes)
         self._add_timeout(env, timeout)
 
-    def _ec_cell_size_variant(self, env, variant):
-        env['EC_CELL_SIZE'] = str(variant)
+    def _expand_env_ec_cell_size(self, env, ec_cell_size):
+        env['EC_CELL_SIZE'] = str(ec_cell_size)
+
+    def _expand_env_cont_rf(self, env, cont_rf):
+        env['CONT_RF'] = str(cont_rf)
+
+    def _verify_env(self, env):
+        """Check that environment vars are not incompatible."""
+        # TODO easy way to verify OCLASS is compatible with num servers?
```

Comment:
I actually have some database-side code to parse an EC oclass to figure out the data and parity shards. I will eventually port that logic to here as a quick sanity check.